### DISCUSSION TITLE
Hel

### DISCUSSION THREAD

**Discussion by NesUSA** [Reycko/FFTAAV#3] [General]
How do i know if my exe file is actually getting converted, it says it is gonna convert but it never does

**Comment by Reycko** [comment]
i know, couldn't get a WaitForExit() working, if it's still not converted to this day, i've just put an image in the readme for that like literally 10 minutes ago

**Comment by NesUSA** [comment]
well i just wanna make app music

**Comment by Reycko** [comment]
does the example help for you to configure?
